cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/TransactionsCommandTest.scala

```diff
@@ -0,0 +1,261 @@
+
+|m|startTime
+|a|The time at which the transaction was started. label:default-output[]
+
+|m|status
+|a|The current status of this transaction (`Terminated`, `Blocked`, `Closing` or `Running`). label:default-output[]
+
+|m|elapsedTime
+|a|This is the time that has elapsed since the transaction was started returned as a duration. label:default-output[]
+
+|m|allocatedBytes
+|a|The number of bytes allocated on the heap so far by this transaction. label:default-output[]
+
+|m|outerTransactionId
+|a|The transaction ID of any outer transaction.
+
+|m|metaData
+|a|This is any metadata associated with the transaction, or empty if there is none.
+
+|m|parameters
+|a|This is a map containing all the parameters used by the query currently executing in this transaction.
+
+|m|planner
+|a|The name of the Cypher planner used to plan the query currently executing in this transaction. For details, see <<cypher-planner, Cypher planner>>.
+
+|m|runtime
+|a|The name of the Cypher runtime used by the query currently executing in this transaction. For details, see <<cypher-runtime, Cypher runtime>>.
+
+|m|indexes
+|a|The indexes utilised by the query currently executing in this transaction.
+
+|m|protocol
+|a|The protocol used by connection issuing the transaction.
+|This is not necessarily an internet protocol (like http et.c.) although it could be. It might also be "embedded" for example, if this connection represents an embedded session.
+
+|m|requestUri
+|a|The request URI used by the client connection issuing the transaction, or null if the URI is not available.
+
+|m|statusDetails
+|a|Provide additional status details from underlying transaction or blank if none is available.
+
+|m|resourceInformation
+|a|Information about what transaction is waiting for when it is blocked.
+
+|m|activeLockCount
+|a|Count of active locks held by the transaction.
+
+|m|cpuTime
+|a|CPU time that has been actively spent executing the query returned as a duration.
+|This field will be null unless the config parameter `dbms.track_query_cpu_time` is set to true.
+
+|m|waitTime
+|a|Accumulated transaction waiting time that includes waiting time of all already executed queries plus waiting time of the currently executed query as a duration.
+|This field will be null unless the config parameter `dbms.track_query_cpu_time` is set to true.
+
+|m|idleTime
+|a|Idle time for this transaction returned as a duration.
+|This field will be null unless the config parameter `dbms.track_query_cpu_time` is set to true.
+
+|m|allocatedDirectBytes
+|a|Amount of off-heap (native) memory allocated by this transaction in bytes.
+
+|m|estimatedUsedHeapMemory
+|a|The estimated amount of used heap memory allocated by this transaction in bytes.
+
+|m|pageHits
+|a|The total number of page cache hits that this transaction performed.
+
+|m|pageFaults
+|a|The total number of page cache faults that this transaction performed.
+||===""")
+      section("Syntax") {
```

Comment:
would it make more sense to have the _syntax_ before the _columns_? (also for terminate)

Reply:
I followed the same order as SHOW FUNCTIONS and SHOW PROCEDURES

Reply:
I guess it is fine then, just suddenly felt it was a weird order 🤷